Userland/Libraries/LibC/arpa/nameser_compat.cpp

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+extern "C" {
+}
```

Comment:
Adding empty files here isn't particularly useful, please remove them.,

Reply:
At some point, something didn't work without these corresponding source files.
I just removed them, and it seems to work fine now.